scripts/main.py

```diff
@@ -9,21 +8,21 @@
 import load.topics as load_topics
 import load.speeches as load_speeches
 import os
-import sys
+from googleapiclient.discovery import build
```

Comment:
````suggestion
import googleapiclient
````
see change in line 46. in general, i prefer to have the full name of the library which makes debugging slightly easier when there's something wrong (that i can search the documentation more easily.